companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "metus-technology-ingest-controller",
```

Comment:
I thought, by convention, these matched part of the repo name, but I've no idea if that matters or is just usual?

Reply:
Yeah, I think my expectation was to rename the module to match this new name, but I was holding off on doing that until merging for some reason